programs/mango-v3-reimbursement/src/instructions/create_reimbursement_account.rs

```diff
@@ -0,0 +1,37 @@
+use anchor_lang::prelude::*;
+
+use crate::state::{Group, ReimbursementAccount};
+
+#[derive(Accounts)]
+pub struct CreateReimbursementAccount<'info> {
+    pub group: AccountLoader<'info, Group>,
+
+    #[account(
+        init,
+        seeds = [b"ReimbursementAccount".as_ref(), group.key().as_ref(), mango_account_owner.key().as_ref()],
+        bump,
+        payer = payer,
+        space = 8 + std::mem::size_of::<ReimbursementAccount>(),
+    )]
+    pub reimbursement_account: AccountLoader<'info, ReimbursementAccount>,
+
+    /// CHECK: address is part of the ReimbursementAccount PDA
+    pub mango_account_owner: UncheckedAccount<'info>,
+
+    #[account (
+        constraint = signer.key() == mango_account_owner.key() || signer.key() == group.load()?.authority
+    )]
+    pub signer: Signer<'info>,
```

Comment:
Is there any issue with making CreateReimbursementAccount permissionless? 

That could allow integrations to manually create ReimbursementAccounts for all their mango account owners. The amount of integration work they need to do would reduce a bit.

I also wonder if we should make it idempotent by using `init-if-needed`. That way even if an integration CPIs this call, they don't need to keep track of whether it was called before.